script/publish_public_data_json

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python3
+
+"""
+Export query result data as JSON to a publicly accessible bucket.
+
+Data of the query is exported if "json_export" is set in
+the corresponding metadata file.
+"""
+
+from argparse import ArgumentParser
+import os
+import sys
+import re
+
+from google.cloud import bigquery
+
+# sys.path needs to be modified to enable package imports from parent
+# and sibling directories. Also see:
+# https://stackoverflow.com/questions/6323860/sibling-package-imports/23542795#23542795
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from bigquery_etl.parse_metadata import Metadata  # noqa E402
+
+
+DEFAULT_BUCKET = "ascholtz-dev-test"  # todo
```

Comment:
Currently, the bucket for public data doesn't exist. I'll open a ticket for this.